core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1471,6 +1471,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey UNDERFS_S3_IN_STREAM_BUFFER_SIZE =
```

Comment:
nit: better remove this property after we did micro bench and decide what's the best size for most cases. We have too many properties...

Reply:
Agree! but i am a little worried about different access patterns and data set may be good for different buffer size. But it will be good if we have a "always close to optimal" buffer size as default and no related property key